src/app/public/modules/flyout/flyout.component.ts

```diff
@@ -61,6 +69,10 @@ let nextId = 0;
   selector: 'sky-flyout',
   templateUrl: './flyout.component.html',
   styleUrls: ['./flyout.component.scss'],
+  providers: [
+    SkyFlyoutMediaQueryService,
+    { provide: SkyMediaQueryService, useExisting: SkyFlyoutMediaQueryService }
```

Comment:
My understanding is that any usage of `SkyMediaQueryService` (by components buried within the flyout) will be overwritten by `SkyFlyoutMediaQueryService`. Is that correct?

If so, I'm confused why this branch does not seem to output the correct media query breakpoint. Since it's in the flyout, I'd assume that the breakpoint returned would be `xs` or `sm`, but it's returning `lg`.

See: https://github.com/blackbaud/skyux-flyout/compare/media-container-addition...media-container-addition-test

Reply:
Correct. We aren't changing anything about the child components and so if they aren't inside a flyout they will still use the "real" service. However, Angular will trace up the tree to find a provider so when inside of a flyout it will find this overwritten provider and thus use the Flyout's version instead.